src/Microsoft.DotNet.Darc/src/Darc/Operations/CloneOperation.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using CSharpx;
```

Comment:
Does this affect source build-ability of this?

Reply:
Removed.